### ISSUE TITLE
Boxes missing from form fields on Ontario rent Increase form

### ISSUE THREAD

**Issue by jrmuizel** [mozilla/pdf.js#14741]
Attach (recommended) or Link to PDF file here:
[N1.pdf](https://github.com/mozilla/pdf.js/files/8401593/N1.pdf)

Configuration:
- Firefox Nightly

What is the expected behavior?

The "On" box and "your rent will increase to" boxes should have a box for each character. They do in Preview, Adobe Reader, and Chrome

What went wrong?
No boxes. The font used also seems different in Chrome/Adobe Reader vs Firefox. (There's no line through the '0' character in Chrome or Firefox)

**Comment by timvandermeij** [comment]
Those fields are comb fields (text widget annotations) and I don't think we do anything with the appearance stream in the display layer. More importantly, we also don't seem to keep it around when printing. I would expect at least printing to keep the original appearance stream, but preferably the display layer should also take it into account.

**Comment by calixteman** [comment]
Yep, the only information we extract from the appearance are the font size, fg color and the font size:
https://github.com/mozilla/pdf.js/blob/7c8a92da77aa658ff72ef2591d59f1d47971ef41/src/core/default_appearance.js#L28
and a new appearance is created when printing/saving for a comb:
https://github.com/mozilla/pdf.js/blob/7c8a92da77aa658ff72ef2591d59f1d47971ef41/src/core/annotation.js#L1852

So if we want to display something from the appearance, we must be able to extract those drawing instructions but without getting ones used to display the text (field value).
